libraries/recipe_helpers.rb

```diff
@@ -10,6 +10,17 @@ def application_key(node)
         run_state_or_attribute(node, 'application_key')
       end
 
+      def is_agent6(node)
+        if run_state_or_attribute(node, 'agent6') == true
+          if run_state_or_attribute(node, 'hybrid') == true
```

Comment:
I know this isn't very idiomatic, but surprisingly, `run_state_or_attribute` was still truthy despite the attribute being `false`